weaver/cli.py

```diff
@@ -2899,7 +2961,6 @@ def main(*args):
     except Exception as exc:
         msg = "Operation failed due to exception."
         err = fully_qualified_name(exc)
-        result = OperationResult(False, message=msg, body={"message": msg, "cause": str(exc), "error": err})
```

Comment:
Was this removed by mistake? 
The CLI should report the error that occurred.

Reply:
yup I saw that in the linting checks. I'll add it back in